robot_skills/src/robot_skills/arms.py

```diff
@@ -22,42 +22,144 @@
 # Temporary: this should be approximately 0.02 once the sensor is correctly setup
 GRASP_SENSOR_LIMITS = tuple(rospy.get_param("skills/arm/grasp_sensor/limits", [0.0025, 0.18]))
 
+
+# Constants for arm requirements. Note that "don't care at all" is not here, as
+# it can be expressed by not imposing a requirement (set it to None).
+
+# Specific types of gripper.
+GRIPPER_TYPE_PINCH = "gripper-type-pinch"
+GRIPPER_TYPE_PARALLEL = "gripper-type-parallel"
+GRIPPER_TYPE_SUCTION = "gripper-type-suction"
```

Comment:
Sorry to nitpick, but I'd like to group these in a Python class per enum (Python 2.7 doesn't feature true Python Enum yet, that was introduced in Python 3.4). 

Eg. change to:
```python
class GripperPrinciple(object):
    Pinch = "pinch"
    Parallel = "parallel"
    Suction = "suction"
```

Reply:
Same goes for those below. Especially since 2 groups are prefixed with the same name.